### PR TITLE
[OVMM] RL improvements

### PR DESCRIPTION
**Pull request by JHurricane96**
## Motivation and Context

- Add camera tilt as configurable option
- Skip panoptic sensor as input to encoder
- Visualize nav goal sensor
- Fix discrete action space: use `step_filter` instead of `snap_point` to prevent sliding, cutting corners, teleporting through thin walls, etc.

## Types of changes

- **\[Docs change\]** Addition or changes to the documentation
- **\[Refactoring\]** Large changes to the code that improve its functionality or performance
- **\[Dependency Upgrade\]** Upgrades one or several dependencies in habitat
- **\[Bug Fix\]** (non-breaking change which fixes an issue)
- **\[Development\]** A pull request that add new features to the [habitat-lab](/habitat-lab) task and environment codebase. Development Pull Requests must be small (less that 500 lines of code change), have unit testing, very extensive documentation and examples. These are typically new tasks, environments, sensors, etc... The review process for these Pull Request is longer because these changes will be maintained by our core team of developers, so make sure your changes are easy to understand!
- **\[Experiment\]** A pull request that add new features to the [habitat-baselines](/habitat-baselines/) training codebase. Experiments Pull Requests can be any size, must have smoke/integration tests and be isolated from the rest of the code. Your code additions should not rely on other habitat-baselines code. This is to avoid dependencies between different parts of habitat-baselines. You must also include a README that will document how to use your new feature or trainer. **You** will be the maintainer of this code, if the code becomes stale or is not supported often enough, we will eventually remove it.

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation if required.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes if required.

### CODE REVIEW

File: habitat-baselines/habitat_baselines/rl/ddppo/policy/resnet_policy.py [hunk 1]
```diff
@@ -148,11 +148,13 @@ def __init__(
     ):
         super().__init__()
         self.no_downscaling = no_downscaling
+        # Goal/privileged information sensors
+        skip_sensors = {ImageGoalSensor.cls_uuid, "robot_head_panoptic"}
```
Comment (ykarmesh) [inline habitat-baselines/habitat_baselines/rl/ddppo/policy/resnet_policy.py:152]: Use UUID if possible

File: habitat-lab/habitat/utils/visualizations/utils.py [hunk 2]
```diff
@@ -220,6 +220,13 @@ def observations_to_image(observation: Dict, info: Dict) -> np.ndarray:
             obs_k = observation[sensor_name]
             if not isinstance(obs_k, np.ndarray):
                 obs_k = obs_k.cpu().numpy()
+            if (
+                sensor_name == "cat_nav_goal_segmentation"
+                and obs_k.shape[2] == 2
+            ):
+                obs_k = np.clip(obs_k[:, :, 0] + obs_k[:, :, 1] * 0.2, 0, 1)[
```
Comment (ykarmesh) [inline habitat-lab/habitat/utils/visualizations/utils.py:227]: What is happening here?

**Comment by ykarmesh** [comment]
Hey Arun, Thanks for creating the PR. I don't see the changes related to the discrete action space. Are you planning another PR for that?

File: habitat-lab/habitat/utils/visualizations/utils.py (hunk 2 above)
Comment (JHurricane96) [reply habitat-lab/habitat/utils/visualizations/utils.py:227]: Visualizing the 2-channel goal segmentation sensor by combining the 2 channels into 1. The target object will be white and the receptacle is gray.

Comment (ykarmesh) [reply habitat-lab/habitat/utils/visualizations/utils.py:227]: Thanks for explaining!

**Review by ykarmesh — APPROVED** [review]

**Pull request merged**